tfb

```diff
@@ -98,9 +98,9 @@ SCRIPT_PATH="$(realpath "$0")"
 SCRIPT_ROOT="$(dirname "$SCRIPT_PATH")"
 
 if ! docker network inspect tfb >/dev/null 2>&1; then
-  docker network create tfb >/dev/null
+    docker network create tfb >/dev/null
 fi
 
 test -t 1 && USE_TTY="-t"
-docker build -t techempower/tfb - < ${SCRIPT_ROOT}/Dockerfile
+docker build -t techempower/tfb - <${SCRIPT_ROOT}/Dockerfile
```

Comment:
This will break the docker build?

Reply:
No, this isnt a broken PR. Works perfectly.

Reply:
Sorry to be a nit but I prefer leaving the space between `<` and the `Dockerfile`. It's both easier to read and matches the docker documentation.

https://docs.docker.com/engine/reference/commandline/build/

![image](https://user-images.githubusercontent.com/1304934/57662145-dfb76280-75a2-11e9-97da-e3488e2f58b6.png)